buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -165,6 +165,7 @@
       "alwaysRun": false,
       "displayName": "packages -> dotnet.myget.org",
       "timeoutInMinutes": 0,
+      "condition": "and(succeeded(), eq(variables['PublishToMyGet'], 'True'), eq(variables['PB_ConfigurationGroup'], 'Release'))",
```

Comment:
this seems correct but have you tested it?  The Conditional Task Expression docs don't ever show > 2 clauses in an and(), so it might require nested and()s.  (I sure hope not, we will find out soon enough if so)